examples/neural_quantum_states/src/data/pubchem_IDs/ID_data.bak

```diff
@@ -0,0 +1,32 @@
+'H2', (0, 6)
```

Comment:
what is this file ? usually `.bak` is a backup, this file looks a lot like `ID_data.dir`

Reply:
The code stores some basic data (names attached to pubchem IDs) in memory using the shelve module, and these files are generated as a result of that. Truthfully, I do not know if they are all necessary.